invisible_cities/reco/xy_algorithms.py

```diff
@@ -58,100 +70,119 @@ def count_masked(cs, d, datasipm, is_masked):
-
-                ***If lm_radius is set to a negative number, the algorithm will simply return
-                the overall barycenter all the SiPms above threshold.***
-
-
-                ---------
-                    This kwarg has some physical motivation. It exists to try to partially
-                compensate problem that the NEW tracking plane is not continuous even though light
-                can be emitted by the EL at any (x,y). When lm_radius < pitch, the search for SiPMs
-                that might contribute pos and charge to a new Cluster is always centered about
-                the position of hottest_sipm. That is, SiPMs within new_lm_radius of
-                hottest_sipm are taken into account by barycenter(). In contrast, when
-                lm_radius = pitch or pitch*sqrt(2) the search for SiPMs contributing to the new
-                cluster can be centered at any (x,y). Consider the case where at a local maximum
-                there are four nearly equally 'hot' SiPMs. new_local_maximum would yield a pos,
-                pos1, between these hot SiPMs. Searching for SiPMs that contribute to this
-                cluster within new_lm_radius of pos1 might be better than searching searching for
-                SiPMs  within new_lm_radius of hottest_sipm.
-                    We should be aware that setting lm_radius to some distance greater than pitch,
-                we allow new_local_maximum to assume any (x,y) but we also create the effect that
-                depending on where new_local_maximum is, more or fewer SiPMs will be
-                within new_lm_radius. This effect does not exist when lm_radius = 0
-                    lm_radius can always be set to 0 mm, but setting it to 15 mm (slightly larger
-                than 10mm * sqrt(2)), should not hurt.
-
-    new_lm_radius = radius, find a new cluster by calling barycenter() on pos/qs of SiPMs within
-                    new_lm_radius of new_local_maximum
-
-    consider_masked  = true if masked SiPMs are considered
-
-    returns
-    c    : a list of Clusters
-
-    Usage Example
-    In order to create each Cluster from a 3x3 block of SiPMs (where the center SiPM has more
-    charge than the others), one would call:
+    Creates a list of clusters with the following steps:
+    - identifying the SiPM with highest charge (which must be > `Qlm`)
+    - obtaining the barycenter from the SiPMs within `lm_radius` of
+      the SiPM with highest charge to locate the local maximum of the
+      charge distribution
+    - obtaining the barycenter from the SiPMs within `new_lm_radius`
+      of the new local maximum
+    - creating a cluster from the SiPMs neighbouring the barycenter of
+      the new local maximum, provided it contains at least `msipm`
+    - discarding (non-destructively) the sipms that formed the cluster
+    - repeating the previous steps until there are no more SiPMs with
+      charge higher than `Qlm`
+
+    Parameters
```

Comment:
Why did you change `arguments` for `Parameters`? Aren't they the arguments of the function?

Reply:
Ugh, I always get confused with this, but:
A parameter is a variable in a function definition. An argument is the actual value.

Besides, in general, I tried to replicate the style that I started to implement for this repo (see for example the core submodule), which is pretty much like numpy's.